### PR TITLE
Java & Android SDK logging examples added.

### PR DESCRIPTION
**Pull request by novalisdenahi**
### Description

Java & Android SDK logging examples added to the documentation.

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [x] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

**Review by z4kn4fein — APPROVED** [review]

File: website/docs/sdk-reference/android.md [hunk 1]
```diff
@@ -593,6 +593,12 @@ Evaluating rule: [Email:john@example.com] [CONTAINS] [@something.com] => no matc
 Evaluating rule: [Email:john@example.com] [CONTAINS] [@example.com] => match, returning "true"
 ```
 
+### Logging Implementation
+
+You can use any slf4j implementation for logging. Some available logger implementations doesn't show debug level messages by default. In these cases, you have to set up the logger configuration to receive all log message from the ConfigCat SDK.
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/android.md:598]:
```suggestion
You have the flexibility to use any slf4j implementation for logging with ConfigCat. However, some logger implementations may not display debug level messages by default. In these cases, you simply need to adjust the logger configuration to receive all log messages from the ConfigCat SDK.
```

File: website/docs/sdk-reference/android.md [hunk 2]
```diff
@@ -593,6 +593,12 @@ Evaluating rule: [Email:john@example.com] [CONTAINS] [@something.com] => no matc
...
+
+You can find <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-kotlin/app/src/android-logger.properties" target="_blank">android-logger</a> and <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-java/app/src/main/assets/logback.xml" target="_blank">logback</a> configuration examples in the [Sample Apps](#sample-apps).
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/android.md:600]:
```suggestion
Examples for <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-kotlin/app/src/android-logger.properties" target="_blank">android-logger</a> and <a href="https://github.com/configcat/android-sdk/blob/master/samples/android-java/app/src/main/assets/logback.xml" target="_blank">logback</a> are available under the [Sample Apps](#sample-apps) section.
```

File: website/docs/sdk-reference/java.md [hunk 3]
```diff
@@ -704,6 +704,12 @@ Evaluating rule: [Email:john@example.com] [CONTAINS] [@something.com] => no matc
 Evaluating rule: [Email:john@example.com] [CONTAINS] [@example.com] => match, returning "true"
 ```
 
+### Logging Implementation
+
+You can use any slf4j implementation for logging. Some available logger implementations doesn't show debug level messages by default. In these cases, you have to set up the logger configuration to receive all log message from the ConfigCat SDK.
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/java.md:709]: same as above

File: website/docs/sdk-reference/java.md [hunk 4]
```diff
@@ -704,6 +704,12 @@ Evaluating rule: [Email:john@example.com] [CONTAINS] [@something.com] => no matc
...
+
+You can find <a href="https://github.com/configcat/java-sdk/blob/master/samples/console/src/main/resources/simplelogger.properties" target="_blank">slf4j-simple</a> and <a href="https://github.com/configcat/java-sdk/blob/master/samples/web/src/main/resources/logback.xml" target="_blank">logback</a> configuration examples in the [Sample Apps](#sample-apps).
```
Comment (sigewuzhere) [inline website/docs/sdk-reference/java.md:711]: same as above

File: website/docs/sdk-reference/android.md (hunk 1 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/android.md]: Fixed, replaced with the suggested text

File: website/docs/sdk-reference/android.md (hunk 2 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/android.md]: Fixed, replaced with the suggested text

File: website/docs/sdk-reference/java.md (hunk 3 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/java.md]: Fixed, replaced with the suggested text

File: website/docs/sdk-reference/java.md (hunk 4 above)
Comment (novalisdenahi) [reply website/docs/sdk-reference/java.md]: Fixed, replaced with the suggested text

**Review by sigewuzhere — APPROVED** [review]

**Pull request merged**